src/node_worker.cc

```diff
@@ -333,7 +333,7 @@ void Worker::Run() {
         HandleScope handle_scope(isolate_);
         InternalCallbackScope callback_scope(
             env_.get(),
-            Local<Object>(),
+            Object::New(isolate_),
             { 1, 0 },
             InternalCallbackScope::kAllowEmptyResource |
```

Comment:
I assume this flag can be dropped from the source tree now?

Reply:
Yep, fixed.